src/styles/styles.js

```diff
@@ -800,6 +800,11 @@ const styles = {
         backgroundColor: themeColors.buttonDefaultBG,
     },
 
+    autoGrowHeightInputContainer: (textInputHeight, maxHeight) => ({
+        height: textInputHeight >= maxHeight ? maxHeight : textInputHeight,
```

Comment:
👋 Coming from https://github.com/Expensify/App/issues/22583
We were missing a check for the minimum height when setting the input height. It only checked if the calculated height is greater than the maxHeight, but didn't consider the case where the calculated height is less than the minHeight.
We resolved this by calculating the height using
```js
lodashClamp(textInputHeight, minHeight, maxHeight)